### PR TITLE
Allow disabling element

### PR DESCRIPTION
**Pull request by camertron**
This PR makes it so `<clipboard-copy aria-disabled="true">` doesn't copy anything when clicked. Because of the way the tests are written, I introduced a new custom event, `clipboard-copy-nothing`, that fires whenever a disabled element is clicked. Definitely open to suggestions around how to avoid a second event 😅 I tried waiting for the promise to time out, but that seems awful.

### CODE REVIEW

**Comment by primer-css** [comment]
:wave: Hello and thanks for pinging us! This issue or PR has been added to our inbox and a Primer first responder will review it soon.
- :art: **If this is a PR that includes a visual change**, please make sure to add screenshots in the description or deploy this code to a lab machine with instructions for how to test.
- :fast_forward: **If this is a PR that includes changes to an interaction**, please include a [video recording](https://support.apple.com/en-us/HT208721) in the description.
- :warning: **If this is urgent**, please visit us in [#primer](https://github.slack.com/archives/CSGAVNZ19) on Slack and tag the first responders listed in the channel topic.

File: src/clipboard-copy-element.ts [hunk 1]
```diff
@@ -8,6 +8,11 @@ async function copy(button: HTMLElement) {
     button.dispatchEvent(new CustomEvent('clipboard-copy', {bubbles: true}))
   }
 
+  if (button.getAttribute('aria-disabled') === 'true') {
+    button.dispatchEvent(new CustomEvent('clipboard-copy-nothing', {bubbles: true}))
```
Comment (keithamus) [inline src/clipboard-copy-element.ts:12]:
```suggestion
```

File: test/test.js [hunk 2]
```diff
@@ -149,6 +153,20 @@ describe('clipboard-copy element', function () {
...
+      target.innerHTML = 'Hello world!'
+      target.id = 'copy-target'
+      document.body.append(target)
+
+      const button = document.querySelector('clipboard-copy')
+      button.setAttribute('aria-disabled', 'true')
+      button.click()
+
+      const text = await whenCopied
+      assert.equal(null, text)
```
Comment (keithamus) [inline test/test.js:168]: Something like this should work:

```suggestion
      let fired = false;
      document.addEventListener('clipboard-copy', () => { fired = true }, { once: true })
      await new Promise(setTimeout)
      assert.equal(fired, false);
      assert.equal(null, text)
```

Comment (keithamus) [inline test/test.js:168]: Something like this should work:

```suggestion
      let fired = false;
      document.addEventListener('clipboard-copy', () => { fired = true }, { once: true })
      
      button.click()
      
      await new Promise(setTimeout)
      assert.equal(fired, false);
      assert.equal(null, text)
```

**Review by keithamus — APPROVED** [review]

**Pull request merged**